cyences_app_for_splunk/default/data/ui/views/cs_mssql.xml

```diff
@@ -8,26 +8,50 @@
         <latest>now</latest>
       </default>
     </input>
-    <input type="text" token="tkn_host" searchWhenChanged="true">
+    <input type="multiselect" token="tkn_host" searchWhenChanged="true">
       <label>Host</label>
       <prefix>host="</prefix>
```

Comment:
I think filter will not apply when selected more than 1 elements. following things to add for all inputs:

```
<prefix>host IN (</prefix>
<suffix>)</suffix>
<valuePrefix>"</valuePrefix>
<valueSuffix>"</valueSuffix>
<delimiter>, </delimiter>
```